.eslintrc.js

```diff
@@ -110,44 +119,19 @@ module.exports = {
       'extends': ['plugin:jest/recommended', 'plugin:jest/style'],
     },
     {
-      'files': ['test/**/*.test.js'],
+      'files': ['test/**/*.test.js', 'test/**/*.test.mjs'],
       'rules': {
+        'import/namespace': 'off',
         'jest/expect-expect': 'off',
         'jest/no-conditional-expect': 'off',
         'jest/no-done-callback': 'off',
-        'jest/no-standalone-expect': 'off',
-        'jest/no-try-expect': 'off',
       },
     },
     {
-      'files': ['test/**/cloudant.v1.test.js'],
+      'files': ['test/**/cloudant.*.test.js', 'test/**/auth-helper.js'],
       'rules': {
+        '@typescript-eslint/no-require-imports': 'off',
         'prettier/prettier': 'off',
-        'object-shorthand': 'off',
```

Comment:
Is this rule not enforced anymore?

Reply:
Same as above, this removes 'off', rule.